agent/Dockerfile

```diff
@@ -1,5 +1,4 @@
-# Install nsenter move to ubuntu:14.04.1
-FROM ibuildthecloud/ubuntu-core-base:14.04-1
+FROM ubuntu:14.04.1
```

Comment:
It's probably better to use `ubuntu:14.04` here, unless you really want to pin to `14.04.1`; if `14.04.2` becomes available, `14.04` will automatically be updated to that, whereas `14.04.1` will not.